backend/inspirehep/records/serializers/bibtex.py

```diff
@@ -4,6 +4,10 @@
 #
 # inspirehep is free software; you can redistribute it and/or modify it under
 # the terms of the MIT License; see LICENSE file for more details.
+import codecs
+import re
+
+import latexcodec
```

Comment:
you need to silence `flake8` here (http://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html#in-line-ignoring-errors).